### ISSUE TITLE
Error including angular element bundle

### ISSUE THREAD

**Issue by matthewfarrell** [pnp/generator-spfx#206]
#### General Information
* [x] Usage
* [ ] Development
* [ ] Documentation
* [ ] Feature Request

#### Specific generator
* [ ] Addons
* [ ] HandlebarsJS
* [x] AngularElements

#### What Version you are running?
* NodeJS: 
* NPM:
* @pnp/spfx: 

#### Describe your problem
The new Angular Elements generator has a bug with the link to the angular bundle file, the generated webpart .ts includes:
```
import 'examples/dist/examples/bundle.js';
```
This should be 
```
import 'examples/dist/examples/bundle';
```

**Comment by StfBauer** [comment]
Thanks @matthewfarrell for the information. Which angular version have you used? can you share your examples projects?

**Comment by matthewfarrell** [comment]
So far I have only tested the generator, I have a couple of existing
Angular 8 projects which I would like to migrate to SharePoint online. I
have to determine if angular routing is going to work before I can do
anything.

Thanks for implementing this BTW, it is going to save me a lot of headache
if I can easily migrate the apps without having to re-write them

**Comment by matthewfarrell** [comment]
Resolved in #207

**Comment by StfBauer** [comment]
Will be published soon in the next release.
